bindings/wasm/examples/public/examples.js

```diff
@@ -502,7 +502,3 @@ for (const [func, code] of Object.entries(examples.functions)) {
       func.replace(/([a-z])([A-Z])/g, '$1 $2');  // Add spaces between words
   examples.functionBodies.set(name, body);
 };
-
-if (typeof self !== 'undefined') {
-  self.examples = examples;
```

Comment:
Intentional? Have you tested ManifoldCAD.org locally?

Reply:
No I haven't yet since I need to install a browser other than Firefox. It wasn't intentional though, as mentioned in the review I added to this block.

Reply:
Firefox should work too, with a small tweak: https://github.com/elalish/manifold/issues/328#issuecomment-1473847102

Reply:
That did work, thanks! Maybe we should mention that in a README somewhere.